### ISSUE TITLE
No rule to make target 'install'.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ashish21590/redis#324]
```
What steps will reproduce the problem?
1. Download and unpack the redis-2.0.0.tgz
2. $ make
3. # make install

What is the expected output? What do you see instead?
The README states that make install will copy the redis binaries to 
/usr/local/bin. However, the Makefile doesn't have an install rule.

What version of the product are you using? On what operating system?
Redis 2.0.0. (current featured download), Ubuntu 10.04.

```

Original issue reported on code.google.com by `milan.ce...@gmail.com` on 5 Sep 2010 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
The problem here is simply that I copied (and modified, but not enough) the 
README file from the Redis master release. There is no make install target in 
2.0! I'll fix either the 2.0 README or add a make install target in Redis 2.0.1.

Cheers,
Salvatore
```

Original comment by `anti...@gmail.com` on 6 Sep 2010 at 8:08
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
make install backported in 2.0.0 branch, will be in 2.0.1, thanks
```

Original comment by `anti...@gmail.com` on 9 Sep 2010 at 8:48
- Changed state: **Verified**
